src/library_noderawfs.js

```diff
@@ -82,12 +82,9 @@ mergeInto(LibraryManager.library, {
         // this stream is created by in-memory filesystem
         return VFS.read(stream, buffer, offset, length, position);
       }
-      var seeking = true;
-      if (typeof position === 'undefined') {
-        seeking = false;
-      }
       var bytesRead = fs.readSync(stream.nfd, NODEFS.bufferFrom(buffer.buffer), offset, length, position);
-      if (!seeking) stream.position += bytesRead;
+      // update position marker when non-seeking
+      if (position == null) stream.position += bytesRead;
```

Comment:
also here, comparison should be strict `===` (assuming we are checking for null and exactly null here).

Reply:
Same thing here, can this be `=== undefined`? also one more spot lower down with `position == null`

(sorry to be pedantic, but it's really easier to read code with strict `===` comparisons rather than JS's default non-strict ones)

Reply:
Oops, I missed this one. Thanks!